python/pyspark/pandas/frame.py

```diff
@@ -5500,6 +5500,25 @@ def op(psser: ps.Series) -> ps.Series:
         else:
             return psdf
 
+    def interpolate(
+        self, method: Optional[str] = None, limit: Optional[int] = None
+    ) -> Optional["DataFrame"]:
```

Comment:
I think it always returns a `DataFrame`. Maybe we should change `Optional["DataFrame"]`to `"DataFrame"`

Reply:
Yes. It should be a `"DataFrame"` before we add param `inplace`